tests/e2e/graphql_mock_schemas/test-custom-response-headers.graphql

```diff
@@ -0,0 +1,21 @@
+schema @server(
+    addResponseHeaders: [["x-custom_header_1", "custom-value_1"],["x-custom_header_3", "custom-value 3"]]
```

Comment:
Should be validated while compiling to blueprint.

Reply:
Done